packages/cli/src/__tests__/commands/__snapshots__/Format.test.ts.snap

```diff
@@ -3,12 +3,12 @@
 exports[`format should add a trailing EOL 1`] = `
 generator client {
   provider = "prisma-client-js"
-  output = "***"
+  output   = "../generated/client"
```

Comment:
Is it intended?

Reply:
Yes, I think the masking of output is only interesting for the integration-tests snapshots